apps/site/pages/docs/fastore-config-options/custom-cypress-test.mdx

```diff
@@ -0,0 +1,43 @@
+---
+title: 'enableCypressExtension'
+sidebar_label: 'faststore.config.js options'
+---
+
+import { Callout, Steps } from 'nextra-theme-docs'
+
+<header>
+
+# `enableCypressExtension` 
+
+</header>
+
+<Callout type="warning" emoji="⚠️">
+This is an experimental feature.
+</Callout>
+
+The `enableCypressExtension` feature flag allows you to utilize Cypress extensions within your FastStore project for running customized Cypress tests seamlessly.
+
+Cypress extensions enhance the testing capabilities of your Faststore project by enabling the incorporation of custom Cypress tests alongside the default tests provided by Faststore.
+
+By default, Faststore includes standard integration tests for your store and with `enableCypressExtension`, you can integrate custom integration tests into a dedicated Cypress root folder within your store's codebase. 
+This integration ensures that both the conventional integration tests and your personalized tests are executed seamlessly as part of the continuous integration (CI) pipeline.
+
+
+## Step 1: Enabling `enableCypressExtension` 
+
+1. Open the `faststore.config.js` file.
+2. Find the `enableCypressExtension` paramether and set it to `true`.
```

Comment:
We'll not let the `experimental` object there as default, it's like a "secret feature". Can we change that, please? 😁 They need to put the object `experimental` and then `enableCypressExtension` to true.